classes/BadpwFailedLogins.inc.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @filesource plugins/generic/betterPassword/classes/BadpwFailedLogins.inc.php
+ * Description of BadpwFailedLogins
+ *
+ * @class BadpwFailedLogins
+ * @brief Container for badPassword failed logins 
+ */
+class BadpwFailedLogins extends DataObject {
+	
+	private $username, $count, $lastLoginTime;	
+	
+	/**
+	 * Constructor
+	 * @param String $username Username
```

Comment:
Capitalized `String` and `Integer` would indicate an object whose class name was `String` or `Integer`.  These are better noted as the primitive types `string` and `int`.